### PR TITLE
Actually enroll people in FAP :)

### PR DESCRIPTION
**Pull request by jessieay**
* This tricky little optional checkbox needs to be checked to enroll
members in food assistance.

screen shot 2017-09-27 at 3 22 02 pm

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -40,15 +40,15 @@
...
 minimal_mailing.update!(
-  city: "",
-  zip: "",
+  street_address: "123 Main St.",
+  city: "Flint",
+  zip: "12345",
   county: "Genesee",
   state: "MI",
-  mailing: false,
+  mailing: true,
```
Comment (jessieay) [inline db/seeds.rb]: updated this to make it more in line with reality: we get a mailing address for all people, but a residential address for some only. 

We validate all fields are present on a mailing address

File: lib/mi_bridges/driver.rb [hunk 2]
```diff
@@ -142,7 +142,6 @@ def debug(e)
       # rubocop:disable Debugger
       binding.pry if ENV["DEBUG_DRIVE"]
       # rubocop:enable Debugger
-      raise e
```
Comment (jessieay) [inline lib/mi_bridges/driver.rb]: by not re-raising we can continue to step through the app even if a debugger statement arises. Is that problematic?

**Comment by jessieay** [comment]
[Check necessary boxes for FAP](https://trello.com/c/UEqmh1V5/360-check-necessary-boxes-for-fap)

File: lib/mi_bridges/driver.rb (hunk 2 above)
Comment (jayroh) [reply lib/mi_bridges/driver.rb]: I would suggest, instead, something like

```ruby
if ENV["DEBUG_DRIVE"]
  binding.pry
else
  raise e
end
```

That way we can continue only if we're in debug mode. Which I think is fair.

Comment (jessieay) [reply lib/mi_bridges/driver.rb]: done!

**Pull request merged**